examples/with-expo/hooks/useAuth.ts

```diff
@@ -15,6 +15,7 @@ import {
 export default function useAuth() {
```

Comment:
this is the wrong file (it's from the example, but the official hooks are in the root folder). I'll update it.

Reply:
oops, my bad hehe